engine/src/flutter/shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformViewsChannel.java

```diff
@@ -306,6 +311,16 @@ public interface PlatformViewsHandler {
      */
     void createForPlatformViewLayer(@NonNull PlatformViewCreationRequest request);
 
+    /** Returns true if support for Hybrid Composition++ Platform Views is enabled. */
+    boolean isHcppEnabled();
+
+    /**
+     * The Flutter application would like to display a new Platform View using Hybrid Composition++.
```

Comment:
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-useless-documentation